functions/packages.bash

```diff
@@ -141,473 +180,467 @@ homegear_setup() {
+    uncomment "#REDIR" /etc/nginx/sites-enabled/openhab
+    comment "listen" /etc/nginx/sites-enabled/openhab
+    uncomment "#SSL" /etc/nginx/sites-enabled/openhab
   fi
+
+  if ! {
+    nginx -t
+    systemctl restart nginx.service
+  }; then echo "FAILED (Nginx service)"; fi
+
+  whiptail --title "Operation Successful!" --msgbox "Setup successful. Please try entering $protocol://$domain in a browser to test your settings." 8 80
 }
 
 ## Function for installing Telldus Core service for Tellstick USB devices.
 ## The function can be invoked either INTERACTIVE with userinterface or UNATTENDED.
 ##
-##    tellstick_core_setup()
+##    telldus_core_setup()
 ##
+telldus_core_setup() {
+  local dist
+  local introtext
+  local successtext
 
-tellstick_core_setup() {
-  FAILED=0
-  introtext="This will install tellstick core services to enable support for USB connected Tellstick and Tellstick duo. For more details, have a look at http://developer.telldus.se/"
-  failtext="Sadly there was a problem setting up tellstick core. Please report this problem in the openHAB community forum or as a openHABian GitHub issue."
-  successtext="Success, please reboot your system to complete the installation.
-
-Next, add your devices in /etc/tellstick.conf.
-To detect device IDs use commanline tool: tdtool-improved --event
-When devices are added restart telldusd.service by using: sudo systemctl restart telldusd
-or just reboot the system.
-"
+  introtext="This will install Telldus Core services to enable support for USB devices connected Tellstick and Tellstick duo."
+  successtext="Success, please reboot your system to complete the installation.\\n\\nNext, add your devices in /etc/tellstick.conf.\\n\\nTo detect device IDs issue the command:\\n- tdtool-improved --event\\n\\nWhen devices are added, restart telldusd.service by rebooting the system or using:\\n- sudo systemctl restart telldusd.service"
+
+  echo -n "$(timestamp) [openHABian] Beginning setup of Telldus Core... "
   if [ -n "$INTERACTIVE" ]; then
-    if ! (whiptail --title "Description, Continue?" --yes-button "Continue" --no-button "Back" --yesno "$introtext" 15 80); then echo "CANCELED"; return 0; fi
+    if ! (whiptail --title "Telldus Core installation?" --yes-button "Continue" --no-button "Cancel" --yesno "$introtext" 8 80); then echo "CANCELED"; return 0; fi
   fi
-  echo -n "$(timestamp) [openHABian] Installing tellstick-core... "
 
   if is_aarch64 ; then
     dpkg --add-architecture armhf
   fi
 
   # Maybe add new repository to be able to install libconfuse1
+  echo -n "$(timestamp) [openHABian] Adding libconfuse1 repository to apt... "
   if is_xenial ; then
     echo 'APT::Default-Release "xenial";' > /etc/apt/apt.conf.d/01release
     echo "deb http://archive.ubuntu.com/ubuntu/ bionic universe" > /etc/apt/sources.list.d/ubuntu-artful.list
-    cond_redirect apt-get update
-    cond_redirect apt-get -y -t=bionic install libconfuse1
+    dist="bionic"
   fi
   if is_jessie ; then
```

Comment:
jessie no longer supported

Reply:
OK so I am clear to remove that? I left it because it was already there.

Reply:
We agreed and documented to support OS versions "n" and "n-1", buster and stretch these are.
It is a still open (and "ever-active") task to remove outdated code but I think we have not opened any separate issue for it.
Also keep #926 in mind, but Ubuntu support is something we should not look at now. Let's get the tidying-up and the new image finalized first.

Reply:
just noticed there's references to jessie and other unsupported releases in openhabiabian.conf files and helpers.bash
We should keep the code in helpers.bash but change all other occurences.
Keep in mind the "n-1" statement will apply to Ubuntu, too, so we will not (once we do) support Ubuntu releases other than "focal" and "bionic".
So if you edit those files anyway remove any rest wherever you see it.